dts/arm/nxp/nxp_k6x.dtsi

```diff
@@ -292,22 +292,52 @@
 			prescaler = <2>;
 		};
 
-		pwm0: pwm@40038000{
-			compatible = "nxp,k64f-pwm";
+		ftm0: ftm@40038000{
+			compatible = "nxp,kinetis-ftm";
 			reg = <0x40038000 0x98>;
+			interrupts = <42 0>;
+			label = "PWM_0";
 			prescaler = <2>;
 			period = <1000>;
 			clock-source = <0>;
 			/* channel information needed - fixme */
+			status = "disabled";
 		};
 
-		pwm1: pwm@40039000{
-			compatible = "nxp,k64f-pwm";
+		ftm1: ftm@40039000{
+			compatible = "nxp,kinetis-ftm";
 			reg = <0x40039000 0x98>;
+			interrupts = <43 0>;
+			label = "PWM_1";
+			prescaler = <2>;
```

Comment:
lets remove prescalar, period, and clock-source if they aren't used and you aren't specing them in the yaml